src/ballet/siphash13/fd_siphash13.h

```diff
@@ -0,0 +1,83 @@
+#ifndef HEADER_fd_src_ballet_siphash13_fd_siphash13_h
+#define HEADER_fd_src_ballet_siphash13_fd_siphash13_h
+
+/* fd_siphash13 provides APIs for SipHash1-3.
+   (1 compression round, 3 finalization rounds) */
+
+#include "../fd_ballet_base.h"
+
+#define FD_SIPHASH13_ALIGN     (128UL)
```

Comment:
Curious about the ALIGN and FOOTPRINT here. I know we use 128 for concurrent data to prevent false sharing, but this looks very single threaded. Is there a perf/other benefit to aligning 128 rather than `alignof( fd_siphash13_private )` ?